app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -55,6 +55,8 @@ public function handle(Rogue $rogue)
             }
 
             if (isset($user)) {
+                $this->updateNorthstarStatus($user, $this->translateStatus($this->record['Status'], $this->record['Finish with State']));
+
```

Comment:
@katiecrane I could be totally missing something but I think `updateNorthstarStatus` only takes two params?

Reply:
@chloealee this is two params! `$user` and `$this->translateStatus($this->record['Status'], $this->record['Finish with State'])`

Reply:
omg my old eyes!